Resources/Prototypes/_LostParadise/Maps/kerberos.yml

```
@@ -0,0 +1,77 @@
+          mapNameTemplate: '{0} NS Керберос {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/_LostParadise/Shuttles/LP_Kerberos_evac.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+            # Service
+            Bartender: [ 2, 2 ]
```

yamllint

[error] too many spaces inside brackets

(brackets)